src/pubhub/pub_hub.bal

```diff
@@ -113,6 +144,14 @@ function startHubAndRegisterTopic() returns websub:WebSubHub {
                                 retryConfig: {
                                     count:  3,
                                     intervalInMillis: 5000
+                                },
+                                followRedirects: { enabled: true, maxCount: 5 },
+                                timeoutInMillis: 5*60000, // Check
+                                secureSocket: {
```

Comment:
Same comment as above. We should be able to reuse the same method, passing the keys as arguments.